src/lib/progress-spinner/progress-spinner.ts

```diff
@@ -144,7 +146,8 @@ export class MatProgressSpinner extends _MatProgressSpinnerMixinBase implements
 
   constructor(public _elementRef: ElementRef,
               platform: Platform,
-              @Optional() @Inject(DOCUMENT) private _document: any) {
+              @Optional() @Inject(DOCUMENT) private _document: any,
+              @Optional() @Inject(ANIMATION_MODULE_TYPE) public _animationMode: string) {
```

Comment:
The `_animationMode` param should also be marked as optional, in case anybody was extending the component.